tests (node 7.7.1)

The following leaks were detected:__core-js_shared__
npm ERR! Test failed.  See above for more details.

Seems to be an issue for some versions of node. In Lab issues it is suggested to add --leaks to the test invocation

`    "test": "lab -T node_modules/lab-babel test --leaks",`

Also 
`lab.test('It requires verified',`... is failing to find `expect(err.errors.verified).to.exist;`. Replacing that with `expect(err.errors.type.message).to.exist;` fixes that test. 

Should I do a pull request?

--- a/src/package.json
+++ b/src/package.json
@@ -1,10 +1,11 @@
+
 {
   "name": "redadalertas-api",
   "version": "1.0.0",
   "description": "",
   "main": "server.js",
   "scripts": {
-    "test": "lab -T node_modules/lab-babel test",
+    "test": "lab -T node_modules/lab-babel test --leaks",
     "start": "nodemon ./server.js --exec babel-node",
     "lint": "eslint .",
     "bootstrap": "babel-node ./scripts/bootstrap"

--- a/src/test/shared/models/test_raid.js
+++ b/src/test/shared/models/test_raid.js
@@ -68,7 +68,7 @@ lab.experiment('Raid model', () => {
     const raid = Raid();
     raid.verified = undefined;
     raid.validate((err) => {
-      expect(err.errors.verified).to.exist;
+      expect(err.errors.type.message).to.exist;
       done();
     });
   });

